docs/articles/documentation/continuous-integration/jenkins.md

```diff
@@ -53,30 +150,49 @@ This runs TestCafe tests from the `tests` directory in Google Chrome. Test resul
 
 ![Run Tests Command](../../images/jenkins/run-tests-command.png)
 
-## Step 4 - Publishing Test Run Reports
+### Configure the Display Server (Linux)
+
+If you run Jenkins on a Linux machine and [use the browser UI](#enable-jenkins-to-start-browser-ui-macos-and-linux), set the `DISPLAY` environment variable to `1` before the `testcafe` command:
+
+```sh
+export DISPLAY=:1
+node_modules/.bin/testcafe chrome tests/**/* -r xunit:res.xml
+```
+
+Alternatively, you can use the [Xvfb server](https://www.x.org/archive/X11R7.6/doc/man/man1/Xvfb.1.xhtml):
```

Comment:
Or you can use built-in headless in Chrome & Firefox, right?

Reply:
Well, I was trying to say that this step applies only when you `[use the browser UI](#enable-jenkins-to-start-browser-ui-macos-and-linux)` - but it seems that it's not so obvious.

Well, since the step is called `Configure the Display Server`, I'd better add a note that it doesn't apply to headless mode, what do you think?

Reply:
Yes, I think we can add such note.